test/e2e/metrics_adapter_test.go

```diff
@@ -31,6 +31,12 @@ import (
 	apiservicesv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
 )
 
+const MetricsServerFeatureGate string = "MetricsServer"
+
+func isMetricsServerFeatureGateEnabled(t *testing.T) bool {
+	return f.IsFeatureGateEnabled(t, MetricsServerFeatureGate)
+}
+
```

Comment:
polish: I think this can be dropped? Just using `f.IsFeatureGateEnabled(t, MetricsServerFeatureGate)` should work the same and is pretty readable too.

Reply:
right, updated :)